hello_app/oauth.py

```diff
@@ -26,6 +28,8 @@
 def register():
    github = oauth.create_client('github')
    redirect_uri = url_for('oauth.authorize', _external=True)
+   app.logger.info(f"Redirecting to: {redirect_uri}")
```

Comment:
Dette skal fjernes ved lejlighed, når vi helt sikkert på det fungere. Det pt. testet på 2 forskellige kontis. Så der stadig lige brug for at beholde det til evt. debug